examples/ccxt.pro/js/binance-watch-ohlcv-many-symbols-continuously.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-const ccxt = require ('ccxt')
```

Comment:
I'm not sure we should do this because both formats are accepted (cjs and esm) so I would like to have examples showcasing both

Reply:
Makes sense..
I created the pr because I would get the following error when calling it from inside the ccxt repo: `ReferenceError: require is not defined in ES module scope, you can use import instead
This file is being treated as an ES module because it has a '.js' file extension and '/home/pablo/github/ccxt/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.`

Maybe we rename to .cjs?
I'm also ok closing the PR if you think thats best.

Reply:
@pcriadoperez I would say we can keep half of them (yes by renaming to `.cjs`) and change the other half to ESM so it's easy to see both options